portfolio/src/main/webapp/faq.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html style="background-image: url('images/spikes.png');">
+  <head>
+    <meta charset="UTF-8">
+    <title>Frequently Asked Questions</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="center">
+      <h1>Frequently Asked Questions</h1>
+    </div>
+    <div id="topnav">
+      <a href="index.html">HOME</a>
+      <a class="active" href="#faq">FAQ</a>
+      <a href="contact.html">CONTACT</a>
+    </div>
+    <div>
+      <ul>
+        <li> Why did you use the Flowerboy album?</li>
+        <p> RNB is my favourite genre and the aesthetic of the Flowerboy album cover is amongst the best I have ever seen.</p>
+        <li> How old are you?</li>
+        <p> I turned 20 on the 1st of June 2020. Being born in 2000 is elite. </p>
+        <li> What is your favourite show? </li>
+        <p> I really enjoyed House of Cards and Black Mirror. Thought-provoking shows always keep me engaged. </p>
+        <li> What is your favourite anime?</li>
+        <p> 
+          There is no solid number one for me, so I'll just say Death Note. The protagonist was somewhat evil, it does not get better
+          than that. 
+        </p>
+        <li> What is your favourite book? </li>
+        <p> 
+          Nervous Conditions by Tsitsi Dangarembga was the first book I ever connected with. Being able to recognise the colloquial
+          language and places where events took place was exciting. It may not seem like a big deal, but a lot of the books and movies
+          we consume in Africa are based in unfamiliar contexts of the Western World.
+        </p>
```

Comment:
Mixing `<li>` and `<p>` inside `<ul>` is strange. Please consider the following pattern:

```
<li><h4> Why did you use the Flowerboy album?</h4>
<p> RNB is my favourite genre and the aesthetic of the Flowerboy album cover is amongst the best I have ever seen.</p></li>
```